handlers/main.yml

```diff
@@ -7,3 +7,8 @@
     name: cadvisor
     state: restarted
     daemon_reload: true
+
+- name: reload systemd units
+  become: true
+  systemd:
+    daemon_reload: true
```

Comment:
I think this is duplicating the `daemon_reload: true` option from the restart handler above which should ensure any configuration change are applied before a restart. 

Have you tested removing the new handler introduced in this PR and simply removing `daemon_reload: true` from the systemd enabling task?

Reply:
I had not tested that, but I suspect that you're right. I have removed the redundant `daemon_reload`.